WebContent/WOH-addRestaurant.jsp

```diff
@@ -0,0 +1,47 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+
+	<link rel="stylesheet" type="text/css" href="site/WOH-site.css">
+	<link rel="stylesheet" type="text/css" href="addRestaurant/WOH-addRestaurant.css">
+	
+	
+	<!-- Latest compiled and minified CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+	
+	<!-- jQuery library -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+	<!-- Latest compiled JavaScript -->
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+	
+	<title>Wheel of Hunger - Add Restaurant</title>
+</head>
+
+<body>
+
+	<nav class="navbar navbar-inverse">
+  	<div class="container-fluid">
+    	<div class="navbar-header">
+      	<a class="navbar-brand" href="#">Wheel of Hunger</a>
+    	</div>
+    	<ul class="nav navbar-nav">
+      	<li><a href="WOH-index.jsp">Home</a></li>
+      	<li class="active"><a href="WOH-addRestaurant.jsp">Add</a></li>
+      	<li><a href="WOH-spinSetup.jsp">Setup Spin</a></li>
+    	</ul>
+    	<ul class="nav navbar-nav navbar-right">
```

Comment:
Personally, I would remove these until we actually implement it.  But that really is just my personal preference.